### PR TITLE
isalist(1) is obsolete: isainfo(1) should be used instead.

### PR DESCRIPTION
**Pull request by mmiyashita**

### CODE REVIEW

**Comment by richsalz** [comment]
@levitte you want to handle this?

File: config [hunk 1]
```diff
@@ -663,7 +663,7 @@ case "$GUESSOS" in
   *-*-linux2) OUT="linux-generic32" ;;
   sun4[uv]*-*-solaris2)
 	OUT="solaris-sparcv9-$CC"
-	ISA64=`(isalist) 2>/dev/null | grep sparcv9`
+	ISA64=`(isainfo) 2>/dev/null | grep sparcv9`
```
Comment (nicowilliams) [inline config]: When was `isainfo` added?  Should this be `(isainfo | isalist) ...`?

Comment (mmiyashita) [inline config]: isainfo(1) was introduced in Solaris 7 in the late 90s, so it should be safe to just replace isalist(1) with isainfo(1).

**Comment by levitte** [comment]
+1

@richsalz , feel free to merge it in

**Comment by richsalz** [comment]
commit 74c2db3 thanks.

**Comment by mmiyashita** [comment]
Thank you!

**Pull request closed without merging**